pkg/admissioncontroller/webhooks/admission/seedrestriction/admission.go

```diff
@@ -89,6 +91,8 @@ func (h *handler) Handle(ctx context.Context, request admission.Request) admissi
 		return h.admitBackupBucket(seedName, request)
 	case backupEntryResource:
 		return h.admitBackupEntry(ctx, seedName, request)
+	case bastionResource:
```

Comment:
Please extend https://github.com/gardener/gardener/blob/c7a65696fa66b7f39bd88698c7e84fcb2331fd5b/charts/gardener/controlplane/charts/application/templates/validatingwebhook-admission-controller.yaml#L99-L107 (for the other resources it'll be done as part of #3957. Similarly, please extend https://github.com/gardener/gardener/blob/c7a65696fa66b7f39bd88698c7e84fcb2331fd5b/hack/local-development/dev-setup-register-gardener#L347-L356.